tests/gis_tests/relatedapp/tests.py

```diff
@@ -304,6 +306,41 @@ def test_collect(self):
         self.assertEqual(4, len(coll))
         self.assertTrue(ref_geom.equals(coll))
 
+    @skipUnlessDBFeature("supports_collect_aggr")
+    def test_collect_filter(self):
+        """
+        Testing the `Collect` aggregate with a filter
+        """
```

Comment:
```suggestion
```
Docstrings & comments that describe the _what_ instead of _why_ are discouraged. The method name should be descriptive enough to understand the problem. 😊

Edit: stupid phone typos 😭